### PR TITLE
Day 10, 14, 16 Jun, Youngsang

### PR DESCRIPTION
**Pull request by yjmark**
## Checklist

- [ x ] You have included a thumbnail image
- [ x ] You have created a new Markdown file to describe your map
- [ x ] You have put _at least_ your name and the map day in the pull request

### CODE REVIEW

File: _maps_2024/10-youngsang-jun.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+day: 10
+creator: Jun, Youngsang
+title: My Childhood Map (Bupyeong-gu, Incheon, Korea)
+thumbnail: /assets/thumbnails/2024/10-youngsang-jun.jpg
+source: https://github.com/yjmark/30DayMapChallenge_2024_Jun/blob/main/Day10_PenandPaper/Day10_Jun.jpg
+---
+
+This paper-and-pen map shows the neighborhood around Buyeong-ro in Bupyeong-gu, Incheon, where I lived during my childhood. The places I remember most are those where spent time with my beloved family and friends, such as playgrounds, schools, and my friends' houses, especially the trampoline in front of the cathedral.
+![Screenshot of interactive map](assets/thumbnails/10-youngsang-jun.jpg)
```
Comment (mjumbewu) [inline _maps_2024/10-youngsang-jun.md:10]: It looks like you're missing the year from this URL here. Same change for your other images.

```suggestion
![Screenshot of interactive map](assets/thumbnails/2024/10-youngsang-jun.jpg)
```

File: _maps_2024/14-youngsang-jun.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+day: 14
+creator: Jun, Youngsang
+title: Do All Countries Use Street Addresses?
+thumbnail: /assets/thumbnails/2024/14-youngsang-jun.jpg
+source: https://github.com/yjmark/30DayMapChallenge_2024_Jun/blob/main/Day14_UPU_StreetAddress/Day14_Jun.Rmd
+---
+
+Not all countries use street names in their addressing systems. Some countries have street names for every road and use them in addresses, while others have few or no street names and rely instead on such as area codes, PO box numbers, or digital geocoding services like what3words.
+![Screenshot of interactive map](assets/thumbnails/14-youngsang-jun.jpg)
```
Comment (mjumbewu) [inline _maps_2024/14-youngsang-jun.md:10]:
```suggestion
![Screenshot of interactive map](assets/thumbnails/2024/14-youngsang-jun.jpg)
```

File: _maps_2024/16-youngsang-jun.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+day: 16
+creator: Jun, Youngsang
+title: Median Household Income and Travel Time to Work Pattern in Three Eastern Cities
+thumbnail: /assets/thumbnails/2024/16-youngsang-jun.png
+source: https://github.com/yjmark/30DayMapChallenge_2024_Jun/blob/main/Day16_ACS_Income-TimeTravel/Day16_Jun.Rmd
+---
+
+How is the relationship between commute time and household income distributed spatially? Today's map examines cases in Philadelphia, Washington D.C., and Boston.
+![Screenshot of interactive map](assets/thumbnails/16-youngsang-jun.jpg)
```
Comment (mjumbewu) [inline _maps_2024/16-youngsang-jun.md:10]:
```suggestion
![Screenshot of interactive map](assets/thumbnails/2024/16-youngsang-jun.jpg)
```

**Review by mjumbewu — CHANGES_REQUESTED** [review]
Just a few small changes requested below.

**Review by mjumbewu — APPROVED** [review]
Looks good to me!

**Comment by bairunnn** [comment]
great work, Jun!

**Pull request merged**